openedx/core/djangoapps/programs/signals.py

```diff
@@ -169,15 +169,11 @@ class of the object instance that sent this signal
 
     # schedule background task to process
     LOGGER.info(
-        'handling COURSE_CERT_REVOKED: username=%s, course_key=%s, mode=%s, status=%s',
-        user,
-        course_key,
-        mode,
-        status,
+        f"handling COURSE_CERT_REVOKED: user={user.id}, course_key={course_key}, mode={mode}, status={status}"
     )
     # import here, because signal is registered at startup, but items in tasks are not yet able to be loaded
     from openedx.core.djangoapps.programs.tasks import revoke_program_certificates
-    revoke_program_certificates.delay(user.username, course_key)
+    revoke_program_certificates.delay(user.username, str(course_key))
```

Comment:
Need to cast `course_key` as a string, a CourseLocator object can't be easily JSON serialized.